### PR TITLE
Parametize docker-compose and standardize Makefile

### PR DESCRIPTION
**Pull request by nbuonin**

### CODE REVIEW

File: drupal-docker.mk [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+docker-clean:
+	-docker stop $(PROJECT)-web
+	-docker stop $(PROJECT)-mysql
+	-docker stop $(PROJECT)-drush
+	-docker rm $(PROJECT)-web
+	-docker rm $(PROJECT)-web
+	-docker rm $(PROJECT)-drush
+
+.PHONY: check-env
```
Comment (sdreher) [inline drupal-docker.mk]: I'm not sure that `make` with no parameters should default to an environment check...that feels a little confusing.

File: sites/default/local_settings.php.sample [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
+# For use for docker-compose local dev. Copy to local_settings.php
+# See drupal-docker.mk for more instructions on setting up local env.
 
 $databases = array (
     'default' =>
     array (
         'default' =>
         array (
-            'database' => 'database',
+            'database' => getenv('PROJECT'),
```
Comment (sdreher) [inline sites/default/local_settings.php.sample]: This is nice.

File: drupal-docker.mk (hunk 1 above)
Comment (nbuonin) [reply drupal-docker.mk]: Oh yeah, I forgot to add a default target.  I'll that in Makefile before the imports.

File: Makefile [hunk 3]
```diff
@@ -4,4 +4,9 @@ PROJECT = classpop
 DC_MYSQL_USER = drupal
 DC_MYSQL_PASSWORD = drupal
 
+.PHONY: info
```
Comment (sdreher) [inline Makefile]: 👍

**Pull request merged**